CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Anyone is welcome to submit PRs and report issues.
+
+## Guidelines for contributions
+
+### Grammar Guidelines
+
+- Any addition to the global grammar will be scrutinized a bit more thoroughly. The more specific a new context, the less scrutiny that is typically applied.
+- To reduce the chance of misrecognitions, is strongly preferred to either (1) match an existing command already in use elsewhere in the repository for consistency or (2) introduce a single command for new functionality.
+  - New grammars should follow the [subject][verb] standard where-ever possible.
+  - Pull requests that mainly introduce alternative voice commands may be rejected. This is necessary to avoid introducing additional pain points for existing users of the repository; changing the grammar without any notice is likely to confusion
```

Comment:
```suggestion
  - Pull requests that mainly introduce alternative voice commands may be rejected. This is necessary to avoid introducing additional pain points for existing users of the repository; changing the grammar without any notice is likely to lead to confusion
```